### PR TITLE
Add Block Class

### PR DESCRIPTION
**Pull request by lokidoki102**
Add Block Class to support add house and seller command.

Add test cases to test for the validity of block number input such as null input and invalid format.

Add a prefix for block to support CLI commands.

### CODE REVIEW

File: src/main/java/seedu/address/model/house/Block.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        requireNonNull(block);
+        checkArgument(isValidBlock(block), MESSAGE_CONSTRAINTS);
+        value = block;
+    }
+
+    /**
+     * Returns true if a given string is a valid block number.
+     */
+    public static boolean isValidBlock(String test) {
+        return test.matches(VALIDATION_REGEX);
```
Comment (KhoonSun47) [inline src/main/java/seedu/address/model/house/Block.java:32]: Looks good, but have you considered that what if the user enter a block that is '0' or '00' or '00A'? I am not too sure as per our requirements can these blocks can be considered as valid because I do not think that a block in Singapore can be '0' in number. But if you done the research and there is block '0', then ignore this comment.

**Review by KhoonSun47 — APPROVED** [review]
LGTM! Maybe you can take note of my comment, but if its not an important requirement, can ignore it. Thanks :)

**Review by felixchanyy — APPROVED** [review]
LGTM
